packages/engine/Specs/Scene/Model/ModelSpec.js

```diff
@@ -715,6 +719,73 @@ describe(
       });
     });
 
+    it("transforms property textures with KHR_texture_transform", function () {
```

Comment:
For new tests, we now prefer to make the spec function `async` and then use `await` rather than promise chains. 

You don't need to update everything in this file (there's a lot of cleanup throughout the codebase), just the specs you added.

Reply:
I'm usually leaning towards some form of "consistency" here (and ... once such a preference is established, would just update everything in one cleanup pass, to prevent others from _also_ following the pattern that they see in the file and of which they don't know that it is not following the latest guidelines). But for now, I only I updated this and the other test.